cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -264,6 +265,10 @@ type ControllerContext struct {
 
 	// Stop is the stop channel
 	Stop <-chan struct{}
+
+	// InformersStarted is closed when the informers have been started by the process.  After this point it is safe,
```

Comment:
Technically we call Start twice. Maybe change it to say we close it after all of the controllers have been initialized and are running?